src/components/tasks/Card/AssigneeButton/AssigneeButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC, HTMLProps } from 'react';
+
+import classNames from '@/components/tasks/Card/AssigneeButton/AssigneeButton.module.scss';
+interface Props extends HTMLProps<HTMLButtonElement> {
+    handleClick: () => Promise<void>;
+    assignee?: string;
+    isDisabled: boolean;
+}
+
+const AssigneeButton: FC<Props> = ({ handleClick, assignee, isDisabled }) => {
+    return (
+        <button
+            className={classNames.card__top__button}
+            type="button"
+            disabled={isDisabled}
+            onClick={handleClick}
+        >
+            {`Assign to ${assignee}`}
```

Comment:
As you have mentioned  ` assignee?: string` where the assignee prop will be optional, in that case, what are we going to show as the Button text?  (`Assign to ${ ??? }`)

Reply:
Its just following the current implementation, if the field is undefined, its gonna print `Assign to undefined` which is inline with the current implementation